migration/idempotent/007-tracing-tags.sql

```diff
@@ -68,19 +68,6 @@ COMMENT ON FUNCTION ps_trace.tag_v_eq(_ps_trace.tag_v, pg_catalog.jsonb)
 IS 'This function is a part of custom _ps_trace.tag_v type which is a wrapper for 
 the built-in jsonb. It is the same as its jsonb_ namesake, but has a support function attached.';
 
-CREATE OR REPLACE FUNCTION ps_trace.tag_v_eq(_ps_trace.tag_v, _ps_trace.tag_v)
```

Comment:
I'm almost tempted to suggest that we keep these function definitions in here (_and_ have them in the `incremental`).

My reasoning is as follows:

In general, we want to have all function definitions in `idempotent`. That way we have one place to look to understand which functions exist and why. In this specific case, we _need_ the function to exist in the `incremental` because we're adding an operator which needs to know about the function, and operators should be added in `incremental` (otherwise we will have a hell of a time managing them).

@sumerman @pgnickb WDYT?

Reply:
I had this thought too but then realized this specific set of functions is translating regular operators 1-1 and is unlikely to change. Therefore I don't have a strong opinion.

Reply:
We used to create a dummy function - the definition with a basically empty body - just prior to creating the operators that depended upon them. Then, in idempotent, we would replace the dummy function with the real body. That way the real code is in one place and in a place where it can be changed easily.

Maintaining the dummy functions was a pain in the ass, so we moved to the pattern of having the operators defined in idempotent as well by wrapping them in a BEGIN/END can catching the exception if they already existed. This way the function and the operator could be kept in the same area of code, and the function is defined in one place where it can be changed.

Why did we decide to move away from this pattern?

Reply:
Good point, thanks for weighing in @jgpruitt. I don't think that there was an explicit decision to move away from that pattern. My original suggestion was to just move the DROP statements to a new incremental, keeping the operator creation in the idempotent script.